Doc/tutorial/classes.rst

```diff
@@ -4,6 +4,12 @@
 Classes
 *******
 
+Classes provide a means of bundling data and functionality together.  Creating
```

Comment:
Reading this, wouldn't the flow be more natural if this was the second paragraph and not the first?